doc/examples/streamline_formats.py

```diff
@@ -145,6 +145,17 @@
 print(rpt_sft.is_bbox_in_vox_valid())
 save_tractogram(rpt_sft, 'rpt.trk')
 
+"""
+Any StatefulTractogram instance consists of clusters of streamlines. Cluster 
+based information such as number of cluster label, streamlines within each cluster
+can be added through a setter function.
+"""
+
+data = {}
+for i in range(len(cc_sft.streamlines)):
```

Comment:
to be more pythonic please use `enumerate` instead `range(len(`

Reply:
Done